v2/pkg/manifest/oci-manifest.go

```diff
@@ -140,10 +142,11 @@ func (o *Manifest) GetRelatedImagesFromCatalogByFilter(filePath, label string, o
 func (o *Manifest) ExtractLayersOCI(fromPath, toPath, label string, oci *v1alpha3.OCISchema) error {
 	if _, err := os.Stat(toPath + "/" + label); errors.Is(err, os.ErrNotExist) {
 		for _, blob := range oci.Layers {
-			if !strings.Contains(blob.Digest, "sha256") {
+			validDigest, err := digest.Parse(blob.Digest)
+			if err != nil {
 				return fmt.Errorf("the digest format is not correct %s ", blob.Digest)
 			}
-			f, err := os.Open(fromPath + "/" + strings.Split(blob.Digest, ":")[1])
+			f, err := os.Open(fromPath + "/" + validDigest.Hex())
```

Comment:
Hi @sherine-k,

It seems the Hex() func is deprecated. It is saying to use Digest.Encoded instead